test/e2e/framework/ingress/ingress_utils.go

```diff
@@ -38,7 +38,7 @@ import (
 	"k8s.io/klog"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
What's the need for this?

Reply:
goimports does this now for performance reasons since the path uses a symlink and the import path isnt the real path.

Ref: https://github.com/kubernetes/kubernetes/pull/76337